go.mod

```diff
@@ -41,3 +41,5 @@ replace (
 	k8s.io/mount-utils => k8s.io/mount-utils v0.21.0
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.0
 )
+
```

Comment:
Do you have a PR open corresponding to your branch here? Is that why you have marked this WIP?

Reply:
Yes I do: https://github.com/jaypipes/ghw/pull/248

Reply:
Ah, great!